tile.yml

```diff
@@ -12,7 +12,7 @@ packages:
   type: app
   manifest:
     memory: 1G
-    buildpack: java_buildpack_offline
+    buildpack: https://github.com/takipi-field/overops-tile.git
```

Comment:
are you sure about this? seems out of scope of the PR, which is for updating the collector version